agent/error.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"syscall"
+)
+
+// isRetryableMountError will check to see if the error passed in is an
+// syscall.EINVAL
+func isRetryableMountError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	errno, ok := err.(syscall.Errno)
+	if !ok {
+		return false
```

Comment:
This can be rewritten slightly shorter. Nil check not needed as in this case `ok` will be just `false`.
```go
func isRetryableMountError(err error) bool
    errno, ok := err.(syscall.Errno)
    return ok && errno == syscall.EINVAL
}
```

Reply:
Fixed